packages/slice-machine/package.json

```diff
@@ -15,8 +15,7 @@
     "postinstall": "node postinstall",
     "prepublishOnly": "yarn bundle",
     "serve": "node -r esm ./build/server/src/index.js",
-    "watch-server": "tsc -w --project server/tsconfig.json",
-    "dev-server": "cross-env PORT=9999 CWD=./tests/project node build/server/src/index.js",
+    "dev-server": "cross-env PORT=9999 CWD=./tests/project ./node_modules/nodemon/bin/nodemon.js -e ts,js --watch server --ignore '*.test.ts' --exec ts-node server/src/index.js",
```

Comment:
Do you have to provide `./node_modules/nodemon/bin/nodemon.js` instead of `nodemon`?

Reply:
Good question, I think we can use nodemon directly

Reply:
Done

Reply:
thank you!